packages/renderer/src/lib/editor/MonacoEditor.svelte

```diff
@@ -64,10 +67,17 @@ onMount(async () => {
     // Emit the content change so we can use it in the parent component
     dispatch('contentChange', editor.getValue());
   });
+
+  isDarkUnsubscribe = isDark.subscribe(value => {
+    isDarkTheme = value;
```

Comment:
You are updating `isDarkTheme` when `isDark` emits a new value, but I don't think that the new value of `isDarkTheme` is used anywhere (the only place it is used is in `onMount`, which won't be run again.